app/src/main/java/app/revanced/integrations/patches/LithoFilterPatch.java

```diff
@@ -214,7 +214,7 @@ public GeneralBytecodeAdsPatch() {
                 //"full_width_square_image_layout",
                 "video_display_full_buttoned_layout",
                 "_ad",
-                "ad_",
+                "ad_badge",
```

Comment:
What about components named ad_something?

Reply:
There are no ad_something based on repo history (only ad_badge).

There was just ```text_search_ad``` and ```text_display_ad```. But for these can be used ```_ad``` instead ```ad_```

Reply:
I posted a screenshot with all ads template names, in the previous conversation.